FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/readme.txt

```diff
@@ -0,0 +1,15 @@
+The subdirectories off this directory contain multiple examples that demonstrate
```

Comment:
grammar: _...subdirectories __of__ this directory..._

Reply:
Fixed.